force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -58,7 +75,34 @@ public with sharing class FlowControlLoopsCollections {
         // Initialize largestNum as null
         Integer largestNum = null;
 
-        return null; // Replace null with the variable you used to store the result
+     //  first check if any of the numbers is null
+        if (num1 == null || num2 == null){
+            
+            return largestNum;
+        }
+        else if ((num1/num2) > 1){
+            largestNum = num1;
+            return largestNum;
+            
+        }
+
+        else if ((num1/num2) < 1){
+            largestNum = num2;
+            return largestNum;
+        }
+
+        else if ((num1 == num2)){
+            largestNum = num1;
+            return largestNum;
+        }
+
+
+        
+
+        else {
+        
+        return largestNum;
```

Comment:
Try to watch the spacing and stick to a style.
Also try to use the Math Method to solve this one.
https://developer.salesforce.com/docs/atlas.en-us.apexref.meta/apexref/apex_methods_system_math.htm#apex_System_Math_max